tools/gn

```diff
@@ -441,6 +441,10 @@ def to_gn_args(args):
     gn_args['is_clang'] = True
   else:
     gn_args['is_clang'] = args.clang
+  if args.slimpeller:
```

Comment:
Should the `gn` script signal any errors if `--slimpeller` is combined with non-metal configs?

Reply:
I didn't really think all flag combinations should be buildable. But it makes sense. Patching.

Reply:
Done.